z390test/src/test/groovy/org/z390/test/RunSortTests.groovy

```diff
@@ -0,0 +1,76 @@
+package org.z390.test
+
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.TestInstance.Lifecycle
+
+
+@TestInstance(Lifecycle.PER_CLASS)
+class RunSortTests extends z390Test {
+
+    var options = [
+            'noloadhigh',
+            "SYSMAC(${basePath("mac")})",
+            "SYSCPY(${basePath("mac")})",
```

Comment:
Remove trailing comma???

Reply:
OK, will remove.. although given it is valid in Groovy I often leave them in in case I need to add additional entries.. Python has the same behaviour.